package.json

```diff
@@ -17,12 +17,13 @@
   },
   "homepage": "https://github.com/lblod/delta-consumer#readme",
   "dependencies": {
-    "@lblod/mu-auth-sudo": "0.6.1",
+    "@lblod/mu-auth-sudo": "0.6.2",
     "cron": "^1.8.2",
     "fs-extra": "^9.0.1",
     "lodash": "^4.17.21",
-    "n3": "^1.25.1",
+    "n3": "^1.26.0",
     "node-fetch": "^2.6.1",
-    "sparqljs": "^3.7.1"
+    "sparqljs": "^3.7.1",
+    "split2": "^4.2.0"
```

Comment:
I didn't find the split2 dependency used at first sight, so might be removed? If not I'd consider just inlining it, since the pacakge hasn't been updated for a while and it's very basic: https://github.com/mcollina/split2/blob/master/index.js